code/game/machinery/telecomms/machines/message_server.dm

```diff
@@ -15,7 +15,60 @@
 	idle_power_usage = 10
 	active_power_usage = 100
 	armor = list("melee" = 25, "bullet" = 10, "laser" = 10, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 70)
+	var/obj/item/stored
 
+/obj/machinery/blackbox_recorder/Initialize()
+	. = ..()
+	stored = new /obj/item/blackbox(src)
+
+/obj/machinery/blackbox_recorder/attack_hand(mob/living/user)
```

Comment:
Might be a good idea to make this only removable by traitors like the nuke core to prevent people stealing the box and hiding it.

Reply:
This can be said about all theft objectives (Jetpack, blueprints) but rarely seems to be an issue. If someone is deliberately going out of their way to hide theft objectives at the start of the round I'd call it an administrative issue. However, if you think it's a better idea, I can add a check.

Reply:
I kind of see it somewhat different for items that aren't useable by people like the nuke core, SM shard rather than the blueprints or X-01

Reply:
The original PR comments mentioned implementing an interesting, if possibly overcomplicated/not worth the trouble feature of having medical and security records 'stored' on the black box; if you pull it, you lose access to those records. That would prevent hiding it, and give a reason to protect it other than "gotta stop the antags".